base/src/main/java/io/fabric8/launcher/base/identity/TokenIdentity.java

```diff
@@ -24,7 +24,6 @@ default void accept(IdentityVisitor visitor) {
     }
 
     static TokenIdentity fromBearerAuthorizationHeader(String authorizationHeader) {
-        requireNonNull(authorizationHeader, "authorizationHeader must be specified.");
```

Comment:
it was there to separate NPE when arg is null from IllegalArgumentException when it's incorrect.